package.json

```diff
@@ -48,6 +48,7 @@
   ],
   "dependencies": {
     "dotenv": "^8.2.0",
+    "emotion-theming": "^10.0.27",
```

Comment:
Hi, I think that `emotion-theming` is only used in the `React` part (client) of your app. I think this can line can be removed here and moved to the package.json in the client folder.

Reply:
Thank you! Yes, the dark mode theme will be a feature. I moved the line in the client folder. Happy Easter!